framework/console/controllers/BaseMigrateController.php

```diff
@@ -678,10 +678,14 @@ public function actionCreate($name)
             }
 
             if ($this->newFileMode !== null) {
-                @chmod($file, $this->newFileMode);
+                if (!@chmod($file, $this->newFileMode)) {
+                    throw new Exception('Unable to change file mode of "' . $file . '" to "' . $this->newFileMode . '"');
+                }
             }
             if ($this->newFileOwnership !== null) {
-                FileHelper::changeOwnership($file, $this->newFileOwnership);
+                if (!@FileHelper::changeOwnership($file, $this->newFileOwnership)) {
```

Comment:
Don't use `@` here

Reply:
If we remove the `@` here, should it be added to the `chown` and `chgrp` functions in `FileHelper::changeOwnership()`?